mne/io/egi/tests/test_egi.py

```diff
@@ -288,4 +292,37 @@ def test_io_egi_crop_no_preload():
     assert_allclose(raw._data, raw_preload._data)
 
 
+@pytest.mark.parametrize('idx, cond, signals, bads', [
```

Comment:
Prevent this from bombing on CIs that have minimal dependencies
```suggestion
@requires_version('mffpy')
@pytest.mark.parametrize('idx, cond, signals, bads', [
```